examples/helloworld/main.go

```diff
@@ -4,7 +4,12 @@ import (
 	"context"
 	"fmt"
 	"log"
+)
+
+//go:generate tinygo build -o plugin-morning/morning.wasm -scheduler=none -target=wasi --no-debug plugin-morning/morning.go
```

Comment:
one last ask, can you update the README "Compile a plugin" in this example to suggest go generate like https://github.com/knqyf263/go-plugin/tree/f89dce62c94944431e37e26136d187db0900028f/examples/host-function-library#compile-a-plugin?

Reply:
done

Reply:
I'm not sure, but seems to makefile should be adopted to run 'go generate' for the whole project

Reply:
@dmvolod I think the top-level makefile has duplication with this, but the project isn't consistent enough to rely on that. In any case, I think makefile shouldn't need this as it has a very nice expression language. Maybe only for testing the README instructions..

Reply:
yeah I just looked I think the top-level Makefile is fine and shouldn't be required to use go:generate.

That said, we could add a different GitHub workflow to ensure go generate works (in a PR that makes it actually work on all examples)